python/ray/tests/test_gcs_ha_e2e.py

```diff
@@ -240,6 +238,11 @@ def check_alive(n):
     wait_for_condition(check_alive, timeout=30, n=2)
 
 
+@pytest.mark.skipif(sys.platform != "linux", reason="Only works on linux.")
+def test_redis_cluster_failure(docker_cluster):
+    pass
+
```

Comment:
did you intend to write a test here?

Reply:
Yeah, I should delete this. Initially I want to add it as docker test, but seems not necessary.